lib/shared/widgets/bubble.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:flutter/material.dart';
+
+class Bubble extends StatelessWidget {
+  const Bubble({
+    required this.child,
+    super.key,
+    this.padding = const EdgeInsets.symmetric(horizontal: 60, vertical: 30),
+    this.direction = BubbleDirection.above,
+    this.color = const Color(0xBFFFFFFF),
+    this.triangleSize = 60,
+  });
+
+  final Widget child;
+  final EdgeInsets padding;
+  final BubbleDirection direction;
+  final Color color;
+  final double triangleSize;
+
+  @override
+  Widget build(BuildContext context) {
+    switch (direction) {
+      case BubbleDirection.above:
+        return Column(
+          children: [
+            Container(
+              padding: padding,
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(30),
+                color: color,
+              ),
+              child: child,
+            ),
+            CustomPaint(
+              size: Size(triangleSize, triangleSize / 2),
+              painter: DrawTriangle(
+                color: color,
+                direction: direction,
+              ),
+            ),
+          ],
+        );
+      case BubbleDirection.atRight:
+        return Row(
+          children: [
+            CustomPaint(
+              size: Size(triangleSize / 2, triangleSize),
+              painter: DrawTriangle(
+                color: color,
+                direction: direction,
+              ),
+            ),
+            Container(
+              padding: padding,
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(30),
+                color: color,
+              ),
+              child: child,
+            ),
+          ],
+        );
+    }
+  }
+}
+
+enum BubbleDirection {
+  above,
+  atRight,
+}
+
+class DrawTriangle extends CustomPainter {
```

Comment:
Should we move this to a seperate file?
Personally i dont like to have multiple (long) classes in one file.